config.rb

```diff
@@ -163,4 +163,10 @@ def self.e2e_test?
   # Load Balancer
   optional :load_balancer_service_project_id, string
   optional :load_balancer_service_hostname, string
+
+  # Let's Encrypt
+  # The following are optional because they are only needed in production.
+  # They are not needed in development or test.
+  optional :lets_encrypt_email, string
```

Comment:
pretty sure you mean "acme" here rather than let's encrypt, right? For one, production ubicloud probably can't use let's encrypt per se, though it's reasonable for developers or smaller installs of clover to use it, on rate limiting grounds.